packages/documentation/src/content/docs/telemetry/integrating.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Building on Top of Rafiki Metrics
+---
+
+Rafiki allows for integrating [Account Servicing Entities](/reference/glossary#account-servicing-entity) (ASE) to build their own telemetry solution based on the [OpenTelemetry](https://opentelemetry.io/) standardized metrics format that Rafiki exposes.
+
+In order to do so, the integrating ASE must deploy its own OpenTelemetry collector that should act as a sidecar container to Rafiki. It needs to provide the OpenTelemetry collector's ingest endpoint so that Rafiki can start sending metrics to it.
+
+## Rafiki Telemetry Environment Variables
```

Comment:
Nitpicky, I know, but I think this section would benefit from more explicit descriptions. I was able to discern what they do based on the name of the variable, but I think some readers would benefit from less ambiguity. For example:

```
- `ENABLE_TELEMETRY`: boolean, defaults to `true`. -> - `ENABLE_TELEMETRY`: boolean, defaults to `true`. Enables the telemetry service on Rafiki.
- `OPEN_TELEMETRY_COLLECTOR_URL`: CSV of URLs (e.g., `http://otel-collector-NLB-e3172ff9d2f4bc8a.elb.eu-west-2.amazonaws.com:4317,http://happy-life-otel-collector:4317`). -> - `OPEN_TELEMETRY_COLLECTOR_URL`: CSV of URLs for Open Telemetry collectors (e.g., `http://otel-collector-NLB-e3172ff9d2f4bc8a.elb.eu-west-2.amazonaws.com:4317,http://happy-life-otel-collector:4317`).